examples/ex6.cpp

```diff
@@ -152,8 +152,14 @@ int main(int argc, char *argv[])
    //     flux to get an error indicator. We need to supply the space for the
    //     smoothed flux: an (H1)^sdim (i.e., vector-valued) space is used here.
    FiniteElementSpace flux_fespace(&mesh, &fec, sdim);
-   ZienkiewiczZhuEstimator estimator(*integ, x, flux_fespace);
-   estimator.SetAnisotropic();
+   // ZienkiewiczZhuEstimator estimator(*integ, x, flux_fespace);
+   // estimator.SetAnisotropic();
+
+   LSZienkiewiczZhuEstimator estimator(*integ, x);
```

Comment:
may we should change the interface to be in sync with https://github.com/mfem/mfem/pull/2862

Reply:
I didn't change the interface, just the internals.